README.md

```diff
@@ -76,18 +76,17 @@ Optionally, if you want to grant write access so `git-node` can write comments:
 
 You can also edit the permission of existing tokens later.
 
-After the token is generated, create an rc file with the following content:
-(`~/.ncurc` or `$XDG_CONFIG_HOME/ncurc`):
-
-```json
-{
-  "username": "your_github_username",
-  "token": "token_that_you_created"
-}
+After the token is generated, you can give it to NCU using:
+
+```sh
+ncu-config set username your_github_username
+# It is strongly recommended to encrypt your token. Do not provide it in the CLI,
+# `ncu-config` will prompt you for it.
+ncu-config set -x token
 ```
 
-Note: you could use `ncu-config` to configure these variables, but it's not
-recommended to leave your tokens in your command line history.
+Note: Encryption is available only if you have `gpg` setup on your machine. If
```

Comment:
I think we should just split it into two different instructions with the encryption one prioritized as "Recommended". It's going to catch people off guard to recommend `-x` first only to come back with a note saying they should setup gpg first - that sounds like a pre-requisite that should be mentioned _before_ we suggest the `-x` flag (and also, what if you don't have gpg set up? The doc provides no suggestions, and it is not great to leave new contributors guessing what they should do in that case when they are new to all this and are just trying to set it up).